### ISSUE TITLE
Typo in JSON-LD schema retrieved from https://www.w3.org/ns/pub-context

### ISSUE THREAD

**Issue by Addeventure** [w3c/pub-manifest#249]
Hi!

The schema returned from https://www.w3.org/ns/pub-context contains:
```
        "inLanguage": {
            "@language" : null,
            "@direction": null,
            "@container": "@list",
            "@id": "http://schema.org/inDirection"
        },
``` 
but I think that "@id" should be: "http://schema.org/inLanguage"

**Comment by mattgarrish** [comment]
Yup, that's definitely a typo. There isn't even such a thing as inDirection in schema.org; that's a property we defined very early on in the process.

@iherman I assume we can make this change immediately since it's outside of the specification proper (factoring in you'll need to republish it to wherever that /ns links goes), but should we figure out what the plan is for addressing the other errata in the document?

**Comment by iherman** [comment]
@mattgarrish I would prefer to mark this is an accepted editorial erratum (as @wareid has done) and then discuss on how to move on with an editorial republication ("when", mostly).

**Comment by mattgarrish** [comment]
We can't fix this issue here. I'm going to move this issue to the pub-vocab repository.
